database/emulator-suite.js

```diff
@@ -0,0 +1,26 @@
+
+
+export async function onDocumentReady() {
```

Comment:
nit: none of these functions actually need to be `export async` because they're not used anywhere else and also don't use `await`.

Reply:
Yuck. PR newb...can I fix these in this flow now that you've approved, or can I only merge at this point?

Reply:
@markarndt just commit to the branch and then push to the same branch, the PR automatically follows.